manim/mobject/geometry.py

```diff
@@ -795,6 +814,30 @@ def __init__(self, **kwargs):
 
 
 class ArrowTip(VMobject):
+    r"""Base class for arrow tips.
+
+
+    Examples
+    --------
+
+    Cannot be used directly, only intended for inheritance::
+        >>> tip = ArrowTip()
+        Traceback (most recent call last):
+        ...
+        NotImplementedError: Has to be implemented in inheriting subclasses.
+
+    Instead, use it like this::
```

Comment:
Suggestion: "Instead, use one of the pre-defined ones, or make a custom one like this:"

Reply:
Reworded in 5528cc30a351ff57a5b06f3ab085620148cc2fe8